devices/leviton.js

```diff
@@ -102,10 +105,12 @@ module.exports = [
             exposes.numeric('ballast_power_on_level', ea.ALL).withValueMin(1).withValueMax(254)
```

Comment:
I'm not sure why, but with the current set of settings `ballast_power_on_level` doesn't seem to be sticking well either

Reply:
It definitely changes something for a time, but it's almost like it doesn't want to save on more than a transient basis sometimes (i.e. the state object comes back eventually and it's 0)

Reply:
Yeah, guess so. The behavior seems to be you can set it to anything, but that gets reset to 0 when the light is turned off and set back to 32 when the light is turned on

I swear it was working at one point before I started messing around with it though, ugh...

Reply:
I see, then maybe also remove it from the exposes

Reply:
Done: https://github.com/Koenkk/zigbee-herdsman-converters/pull/4834/commits/3a13fa02049c999b4559e579fd6a0fc61ad0fc30

Left a comment in case I come back to really debug it :)